seedVault/index.ts

```diff
@@ -0,0 +1,81 @@
+export interface SecureStorageAdapter {
+  get(key: string): Promise<string>;
+  set(key: string, value: string): Promise<void>;
+}
```

Comment:
Should this adapter not have the same structure as the common one where the get could return a null or non-promise value?